code/modules/mob/living/carbon/human/human.dm

```diff
@@ -38,7 +38,8 @@
 	. = ..()
 	if(!CONFIG_GET(flag/disable_human_mood))
 		AddComponent(/datum/component/mood)
-	AddElement(/datum/element/flavor_text/carbon)
+	AddElement(/datum/element/flavor_text/carbon, _name = "Flavor Text", _save_key = "flavor_text")
+	AddElement(/datum/element/flavor_text/carbon, _name = "OOC Notes", _addendum = "Put information on ERP/vore/lewd-related preferences here. THIS SHOULD NOT CONTAIN REGULAR FLAVORTEXT!!", _always_show = TRUE, _save_key = "ooc_notes", _dna_carryover = FALSE)
```

Comment:
Sorry for the late reply, but just make another `/datum/element/flavor_text` subtype for other prefs stuff or something, `/datum/element/flavor_text/carbon` is for dna stuff.